ucm2/Qualcomm/x1e80100/x1e80100.conf

```diff
@@ -19,3 +19,12 @@ If.LENOVOSlim7x {
 	}
 	True.Include.7x.File "/Qualcomm/x1e80100/LENOVO-Slim-7x.conf"
 }
+
+If.DellLatitude7455 {
+	Condition {
+		Type RegexMatch
+		String "${var:DMI_info}"
+		Regex "Dell Inc.*(Latitude|Inspiron).*"
```

Comment:
Regex "Dell Inc.*(Latitude|Inspiron-0YWPR3).*"
  }
  True.Include.latitude.File "/Qualcomm/x1e80100/Dell-Latitude-7455.conf"
  
  with the above change I get
  > $ alsaucm listcards                                                                                                                                                                       [±inspiron14p-fix ●]
  0: hw:0
    DellInc.-Inspiron14Plus7441--0YWPR3
  
without ucm file fails to be found